frontend/src/component/application/ApplicationIssues/ApplicationIssues.tsx

```diff
@@ -128,12 +130,25 @@ const FeaturesMissing = ({ features }: IFeaturesMissingProps) => {
                         <ConditionallyRender
                             condition={hasAccess(CREATE_FEATURE)}
                             show={
-                                <StyledLink
-                                    key={feature}
-                                    to={`/projects/default/create-toggle?name=${feature}`}
-                                >
-                                    Create feature flag
-                                </StyledLink>
+                                <ConditionallyRender
+                                    condition={improveCreateFlagFlow}
+                                    show={
+                                        <StyledLink
+                                            key={feature}
+                                            to={`/projects/default?create=true&name=${feature}`}
```

Comment:
oh, cool! Does the name query param work too? 😄 

I'm guessing we're using the default project because it'll always be there, right? Don't know if there is a better way to select the project, but I also don't know if anyone uses this option 🤷🏼

Reply:
In the application list, we do not have project info. Ideally we would have it, but it is separate issue.
About query param, probably need to implement now that you say it.

Reply:
no, the query param works 😄

Reply:
But yeah, I agree. this is a separate issue, which we may or may not wanna tackle later.

Reply:
It went to the default project before too, so I think that's fine.

Reply:
Ah, it does work.